modules/scanning-images.adoc

```diff
@@ -58,80 +58,96 @@ IMPORTANT: If you are using a Red Hat link:https://catalog.redhat.com/software/c
 
 |===
 
-[discrete]
+[id="supported-package-formats_{context}"]
 == Supported package formats
 
 Scanner can check for vulnerabilities in images that use the following package formats:
 
-* yum
-* microdnf
 * apt
```

Comment:
delete

Reply:
the red is showing that they were deleted